[OP-1121] and [OP-1122] MRs (Fix for docker file and .env into .gitignore) (#323)
informatici/openhospital-api: README.md

Make sure you have docker with docker-compose installed, then run the following commands:

> - rename the `dotenv` file in `.env` and set variables as needed (the SHA-256 jwt token is needed)

Reviewer: ```suggestion
- copy `dotenv` file into `.env` and set variables as needed (the SHA-256 jwt token is needed)
```
